### ISSUE TITLE
Wallet wont sync

### ISSUE THREAD

**Issue by Zenmaine** [EmberCoin/Ember#59]
Wallet will not sync please offer support. I have already sent my EMB to the wallet address provided

**Comment by dan91d** [comment]
I have the same problem. Any solution to this issue?

**Comment by Zenmaine** [comment]
After restarting my computer it started to sync. It did take some time for the sync.

**Comment by Drock415** [comment]
@Zenmaine when it synced did all your coins show up?
  and did you send your coins to the address provided by the wallet in the receive tab?

**Comment by Zenmaine** [comment]
@Drock415 Yes and yes. I unlocked the wallet and just left it alone over night. The following day it showed my coins in the wallet. I sent another batch of coins to the wallet and had the same sync problem, but was able to get my coins by just letting the wallet stay on over night.
